modules/virt-creating-configmap.adoc

```diff
@@ -274,37 +274,23 @@ data:
     "<rhv-vm-windows-os>": "windows-template" <2>
 EOF
 ----
-<1> Optional: You can change the value of the `namespace` parameter.
+<1> Optional: You can change the namespace.
 <2> Specify the REST API name of the RHV VM Windows operating system, for example, `"windows_2016x64"`, and the Windows template.
 endif::[]
 
-. Edit the `kubevirt-hyperconverged-operator.v{HCOVersion}.yaml` file:
+. Patch the `vm-import-controller` deployment to apply the new ConfigMap:
```

Comment:
"Patch the `vm-import-controller-config` ConfigMap to apply the new OS mapping ConfigMap:"

Reply:
Fixed